niworkflows/__about__.py

```diff
@@ -64,6 +64,7 @@
     'scipy',
     'seaborn',
     'svgutils',
+    'templateflow>=0.0.3',
```

Comment:
Since we import from `datalad`, I think we should make it an explicit dependency, even if it's covered by `templateflow`.

```suggestion
    'templateflow>=0.0.3',
    'datalad',
```

Reply:
oooh true. I'll change the import if you don't mind.

Reply:
Done